Venmo/src/main/java/com/braintreepayments/api/VenmoApi.java

```diff
@@ -84,7 +84,7 @@ public void onResult(String responseBody, Exception httpError) {
     void createNonceFromPaymentContext(String paymentContextId, final VenmoOnActivityResultCallback callback) {
         JSONObject params = new JSONObject();
         try {
-            params.put("query", "query PaymentContext($id: ID!) { node(id: $id) { ... on VenmoPaymentContext { paymentMethodId userName payerInfo { firstName lastName phoneNumber email externalId userName" +
+            params.put("query", "query PaymentContext($id: ID!) { node(id: $id) { ... on VenmoPaymentContext { paymentMethodId userName payerInfo { firstName lastName phoneNumber email externalId userName " +
```

Comment:
Take it or leave it - maybe just one-line this whole string to avoid any string concat issues in the future

Reply:
Yeah I agree. Ideally we move this into a raw resource [like on PPCP](https://github.com/paypal/Android-SDK/blob/a9633573119f7580a08f3826b3fdff4363f467ce/CorePayments/src/main/res/raw/graphql_query_funding_eligibility.graphql).